geometry/latlon.hpp

```diff
@@ -14,6 +14,7 @@ class LatLon
   static double constexpr kMaxLat = 90.0;
   static double constexpr kMinLon = -180.0;
   static double constexpr kMaxLon = 180.0;
+  static double constexpr kInvalid = -1000.0;
 
   // Default values are invalid.
   double m_lat = kMinLon;
```

Comment:
So put kInvalid here?

Reply:
kInvalid is instead of using zeros as special values. What is wrong here?

Reply:
Everything is ok, I mean put kInvalid instead of current kMinLon initializers.

Reply:
double m_lat = kInvalid;
double m_lon = kInvalid;
?

Reply:
Updated.